Tests/Packet++Test/PacketExamples/Sll2Packet.dat

```diff
@@ -0,0 +1 @@
+080000000000001400010406d2cfc25015ea0000450000288b2240004006599707f99772743f426cdfcc01bba7ab9fbfaaceb5e95010042256310000
```

Comment:
Can you please also attach a pcap/pcapng file that contains this packet?
It's easier to view the packet that way

Reply:
sorry, I forgot to commit pcap file...